### PR TITLE
f-registration@3.11.0 - Remove asterisks from registration page form …

### PR DESCRIPTION
**Pull request by ZsofiaS**
## What's changed

Removed asterisks from registration page form field labels so that it aligns with the apps.

## UI Review Checks
image

- [x] README and/or UI Documentation has been [created|updated]
- [ ] Unit tests have been [created|updated]
- [ ] This code has been checked with regard to [our accessibility standards](http://fozzie.just-eat.com/documentation/general/accessibility/checklist)

## Browsers Tested

- [x] Chrome (latest)
- [ ] Internet Explorer 11
- [ ] Mobile (Please list device/browser – Ideally one iPhone model and one Android)

### List any other browsers that this PR has been tested in:

- [View the Browser Support Checklist](http://fozzie.just-eat.com/documentation/general/browser-support)

### CODE REVIEW

**Review by adammorr — DISMISSED** [review]

**Review by hwoodiwiss — DISMISSED** [review]

File: packages/components/pages/f-registration/src/components/Registration.vue [hunk 1]
```diff
@@ -46,6 +46,7 @@
                     v-model="firstName"
                     name="firstName"
                     required
+                    :isVisuallyRequired=false
```
Comment (xander-marjoram) [inline packages/components/pages/f-registration/src/components/Registration.vue]: I think you'll need quotes around all of these, and props should be in kebab-case please
```suggestion
                    :is-visually-required="false"
```

File: packages/components/pages/f-registration/CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,13 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+v3.11.0
+------------------------------
+*November 11, 2022*
```
Comment (xander-marjoram) [inline packages/components/pages/f-registration/CHANGELOG.md]: Also just noticed this is next week 😄

Comment (xander-marjoram) [inline packages/components/pages/f-registration/CHANGELOG.md]:
```suggestion
*November 4, 2022*
```

**Review by xander-marjoram — APPROVED** [review]

File: packages/components/pages/f-registration/CHANGELOG.md (hunk 2 above)
Comment (ZsofiaS) [reply packages/components/pages/f-registration/CHANGELOG.md]: good spot! My pc is showing US date 🤦‍♀️

**Comment by ashleynolan** [comment]
Looks good. If you look to bump this in CoreWeb, give me a shout as I changed some styles that the registration components bundles by default (and need to check that CoreWeb has them in there instead).

I was going to do this next week and bump my version, but if you look to pull in yours before then let me know 😄

**Review by ashleynolan — APPROVED** [review]

**Pull request merged**